src/main/scala/com/gu/support/workers/model/state/FailureHandlerState.scala

```diff
@@ -0,0 +1,13 @@
+package com.gu.support.workers.model.state
+
+import java.util.UUID
+
+import com.gu.support.workers.model._
+
+case class FailureHandlerState(
```

Comment:
This is specific to monthly-contributions.  At some point we might need to rename it or move it into a different package, but fine for now.